internal/engine/upper_test.go

```diff
@@ -2299,25 +2296,6 @@ func TestDockerComposeBuildCompletedSetsStatusToUpIfSuccessful(t *testing.T) {
 	})
 }
 
-func TestDockerComposeBuildCompletedDoesntSetStatusIfNotSuccessful(t *testing.T) {
```

Comment:
why did this test get deleted?

Reply:
It isn’t testing anything in our code, all it’s testing is the behavior of the fake build and deployer. 

 If we want to enforce that, when we handle a completed build, we don’t record docker compose state if there is an error, that’s a separate question – though it seems moot since the DC BAD Only ever returns empty results along side errors